tasks/GPSR.tex

```diff
@@ -19,18 +14,14 @@ \subsection*{Setup}
 \begin{itemize}[nosep]
     \item \textbf{Locations:}
         \begin{itemize}
-            \item \textbf{Task location:} The task takes place inside the \Arena{}. Commands may require the robot to leave the room. The \Arena{} is in its nominal configuration for this task.
+            \item \textbf{Task location:} The task takes place inside the \Arena{}. The \Arena{} is in its nominal configuration for this task. Commands may require the robot to manoeuvre to any part of the \Arena{}.
             \item \textbf{Start location:} The robot starts outside the \Arena{}. When the door opens, it must navigate towards the \textit{Instruction Point}.
-            \item \textbf{Instruction point:} The robot returns to this point after completing each command to receive the next one.
+            \item \textbf{Instruction point:} The robot returns to this point after completing all of the commands.
         \end{itemize}
     \item \textbf{People:}
         \begin{itemize}
-			\item \textbf{Professional Operator:} Typically the referee, issues standard commands to the robot.
-			\item \textbf{Non-Expert Operator (optional):} A person without robotics expertise (e.g., from the audience). The referee provides the intended command goal to this operator, who then issues the instruction using their own wording.
-
-            For example, the generated command might be \textit{"Bring me a coke from the kitchen."} then the non-expert operator will be told \textit{"The robot should bring you a coke, which is found in the kitchen."}, who then tells the robot \textit{I want a coke. Go to the kitchen and get me one."}
-
-            If the robot consistently fails to understand the non-expert operator (e.g. after two retries), teams can default to a custom operator.
+			\item \textbf{Professional Operator:} The referee issues standard commands to the robot.
+            If the robot consistently fails to understand after three retries, teams can default to a custom operator.
```

Comment:
Three tries in total, so should be three tries or two retries. I think three tries is clearer